### PR TITLE
doc: Add introduction

### PR DESCRIPTION
**Pull request by matthiasbeyer**
Introduction.

@TheNeikos could you please correct my spelling and grammar?

### CODE REVIEW

File: doc/src/01000-intro.md [hunk 1]
```diff
@@ -1,2 +1,59 @@
...
+contact management, calendar, todo-list functionality, etc.
+The program is intended to be used by so-called "power-users", therefor it is a
+commandline application which offers a polished interface to the user and is not
+bound to an editor, e-mail reader, RSS reader, etc. The program uses standards
+as icalendar, vcard, RSS/Atom, Markdown, etc.
+
+The development of this program may adapt other programs. For example, the
+todo-module may be implemented with the "taskwarrior" application, if possible,
+so we try to re-use functionality which is implemented elsewhere and where other
+people but a lot of effort into. We clearly do not want to duplicate
```
Comment (TheNeikos) [inline doc/src/01000-intro.md]: put, not but

File: doc/src/01000-intro.md [hunk 2]
```diff
@@ -1,2 +1,59 @@
...
+## The Problem {#sec:intro:problem}
+
+The problem "imag" wants to solve is rather simple. When the project was
+initiated, there was no PIM-Suite available which
+
+* was for this domain of users ("power-users", "commandline users")
+* contained functionality to link content
+
+The latter point is the bigger one: "imag" wants to offer functionality to the
+user, to link content. This means not only that a contact may be linked to a
```
Comment (TheNeikos) [inline doc/src/01000-intro.md]: 'wants to offer the ability for users to link content.'

**Comment by TheNeikos** [comment]
Other what I annotated it seems good.

**Comment by matthiasbeyer** [comment]
@homu r+

**Comment by matthiasbeyer** [comment]
@homu r+

**Pull request merged**